notebooks/README.md

```diff
@@ -88,4 +88,4 @@ You can open any page of the documentation as a notebook in colab (there is a bu
 
 ## Community notebooks:
 
-More notebooks developed by the community are available [here](community#community-notebooks).
+More notebooks developed by the community are available [here](../../main/docs/source/en/community.mdx).
```

Comment:
This should be the fully link to the doc page as this is converted in a doc page as well and the relative link will then fail. Also, we should keep the anchor.
```suggestion
More notebooks developed by the community are available [here](https:hf.co/docs/transformers/community#community-notebooks).
```